server/scheduler.go

```diff
@@ -35,3 +35,27 @@ func scheduleLeader(cluster *clusterInfo, s Selector) (*regionInfo, *storeInfo,
 
 	return region, source, target
 }
+
+func scheduleStorage(cluster *clusterInfo, s Selector) (*regionInfo, *storeInfo, *storeInfo) {
```

Comment:
s/scheduleStorage/scheduleStore/

can we add an explanation for the three results?